x-pack/plugins/security_solution/cypress/integration/detection_rules/indicator_match_rule.spec.ts

```diff
@@ -485,6 +488,40 @@ describe('indicator match', () => {
           .should('have.text', newThreatIndicatorRule.severity.toLowerCase());
         cy.get(ALERT_RULE_RISK_SCORE).first().should('have.text', newThreatIndicatorRule.riskScore);
       });
+
+      it('Investigate alert in timeline', () => {
+        const indicatorMatchRule = { ...newThreatIndicatorRule };
+        const accessibilityText = `Press enter for options, or press space to begin dragging.`;
+        const indicatorMatchTimelineTemplatePath =
+          'server/lib/detection_engine/rules/prepackaged_timelines/threat.json';
+        const threatIndicatorPath =
+          '../../../x-pack/test/security_solution_cypress/es_archives/threat_indicator/data.json';
+
+        loadPrepackagedTimelineTemplates();
+        cy.readFile(indicatorMatchTimelineTemplatePath).then((template) => {
```

Comment:
It's unfortunate that the prepackaged API does not give back IDs ☹️ . Reading from the file here is okay, but I think it's more likely that the file path will change than the ID of that template changing. If the title doesn't need to be identical, I would just consider declaring the `templateTimelineId` as a constant in this file and using a custom title.

Reply:
In that case, I will put the templateTimelineId as a constant for the indicator match rule object we use at the test :)